src/app/lists/Dataview.tsx

```diff
@@ -62,9 +62,14 @@ const ListsDataView: React.FC<Props> = ({ data: rawData }) => {
         const unmappedTooltips: RowData = {};
 
         for (const [key, value] of Object.entries(row)) {
-            if (value && key.endsWith("quantity")) {
+            if (!value) {
```

Comment:
Sorry... more opinions.... 
I don't really like `continue` -> what do you think about `if (value)` instead?

Reply:
You've been outnumbered...

Reply:
Although I was also outnumbered